# Serie 3

## Aufgabe 3.1
Die monatlichen Aufwendungen X [CHF] für den Wasserverbrauch eines 2-Personenhaushalts seien durch eine Zufallsvariable mit der folgenden Wahrscheinlichkeitsdichtefunktion gegeben:

$$f(x)=\left\{
    \begin{matrix}
    cx(15-\frac{x}{4})\ && \text{falls}\ 0\leq x\leq 60 \\
    0 && \text{sonst}
    \end{matrix}
\right. $$

### a)

Welchen Wert muss $c$ annehmen?

$$\int\limits_0^{60}{cx(15-\frac{x}{4})} = 1 $$

$$\left.\frac{15\cdot c\cdot x^2}{2}-\frac{c\cdot x^3}{12}\right|_0^{60} = 1 $$

$$\frac{15\cdot c\cdot 60^2}{2}-\frac{c\cdot 60^3}{12} = 1 $$

$$\frac{54000\cdot c}{2} - 18000\cdot c = 1 $$

$$27000\cdot c - 18000\cdot c = 1$$

$$c = \frac{1}{9000}$$

In [7]:
print(str(15*60**2))
print(str(60**3/12))

54000
18000.0


### b)

Geben Sie die Verteilungsfunktion $F$ der Zufallsvariablen $X$ an.

$$ F(x)=\left\{
    \begin{matrix}
    0 && \text{falls}\ x<0\\
    \frac{1}{9000}(\frac{15}{2}x^2-\frac{1}{12}x^3)\ && \text{falls}\ 0\leq x\leq 60\\
    1 && \text{falls}\ x>60
    \end{matrix}
\right. $$

### c)

Welcher Wert der Aufwendungen wird nur mit 10% Wahrscheinlichkeit überschritten?

Gesucht ist ein $a$ sodass $P(X>a)=0.1$ also ist $P(X\leq a)=0.9$, somit muss gelten $F(a)=0.9$. Mit der Formel aus b):

$$\frac{1}{9000}(\frac{15}{2}a^2-\frac{1}{12}a^3) = 0.9$$

$$\frac{15}{2}a^2-\frac{1}{12}a^3 = 0.9\cdot 9000$$

$$\frac{15}{2}a^2-\frac{1}{12}a^3 - 8100 = 0\cdot 9000$$

In [10]:
import sympy as sym

a = sym.Symbol('a')
sym.solveset(15/2*a**2-1/12*a**3-8100)

{-28.6249720273366, 48.2519936604545, 70.3729783668821}

Davon liegt nur $a = 48.252$ im Wertebereich von $F(x)$ und ist somit unsere gesuchte Grenze.

### d)

Wie hoch sind die erwarteten monatlichen Aufwendungen für den Wasserverbrauch eines 2-Personenhaushalts?